front/app/containers/Authentication/steps/SSOVerification/index.tsx

```diff
@@ -12,7 +12,7 @@ import claveUnicaButtonMessages from 'components/UI/ClaveUnicaButton/messages';
 
 import { FormattedMessage } from 'utils/cl-intl';
 
-import NemlogInButton from '../_components/NemLogInButton';
+import MitIdButton from '../_components/MitIdButton';
```

Comment:
Decided to merge both Nemlogin and Criipto verification buttons into one - visually they are no different - both are 'Verify with MitID'